test/models/cheatsheet_test.rb

```diff
@@ -0,0 +1,57 @@
+require 'test_helper'
+
+class CheatsheetTest < ActiveSupport::TestCase
```

Comment:
Is Hartl's new version of Rails Tutorial not using RSpec?

Reply:
Not the version I'm using (the online one). Do you recommend I use rspec instead of minitest?

Reply:
Well ... it *seems* like rspec is the more/less standard, but DHH is a minitest fan (but admits he lost the battle 😄 ). So, I'm glad you're getting experience in both. Keep this for this project and after that you'll have seen both and can decide which you like better. As long as you're testing and getting the coverage, I think you'll be fine.